### PR TITLE
bugfix(TESB-29684): update features XML to match the updated tesb-rt-se

### PR DESCRIPTION
**Pull request by jgreffe**

### CODE REVIEW

**Comment by xldai** [comment]
https://github.com/Talend/activemq/blob/tesb/activemq-5.15.5.x/activemq-karaf/src/main/resources/features.xml 
Here the activemq-camel bundle has also to be patched "5.15.5-tesb1".

File: activemq-karaf/src/main/resources/features.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
     <feature name="activemq-camel" version="${upstream.version}" resolver="(obr)" start-level="50">
         <feature version="${upstream.version}">activemq-client</feature>
-        <bundle>mvn:org.apache.activemq/activemq-camel/${upstream.version}</bundle>
+        <bundle>mvn:org.apache.activemq/activemq-camel/${activemq-camel.tesb.version}</bundle>
```
Comment (jgreffe) [inline activemq-karaf/src/main/resources/features.xml:42]: @xldai , version update is done here already

Comment (xldai) [reply activemq-karaf/src/main/resources/features.xml:42]: Yep, my fault.
then LGTM.

**Review by xldai — APPROVED** [review]
LGTM

**Pull request merged**